### ISSUE TITLE
Missing this.getDOMNode()

### ISSUE THREAD

**Issue by maxwellpeterson-wf** [cleandart/react-dart#30]
The component API seems to be missing [`this.getDOMNode()`](http://facebook.github.io/react/docs/component-api.html#getdomnode)

Error:
```
Class '_SettingsModal' has no instance method 'getDOMNode'.
NoSuchMethodError: method not found: 'getDOMNode'
```

**Comment by hleumas** [comment]
Thank you for reporting! We will take a look at this issue soon.

**Comment by hleumas** [comment]
@matystl please publish on pub and then close these issues.

**Comment by matystl** [comment]
implemented in 0.5.2 and published on pub
